config/routes.rb

```diff
@@ -74,6 +74,7 @@
       resources :organizations, only: %i[show index create update destroy] do
         resources :bank_accounts, only: %i[show index create update destroy] do
           get :visible, on: :collection
+          put :sort, on: :member
```

Comment:
@yunixon Rename path to `position` (or anything else that you think will be more suitable). There should be only nouns in a path (action usually is a verb).
```suggestion
          put : position, action: :update_ position, on: :member
```

(I know we have a lot of usage of verbs in paths, but this is wrong and we will fix it in future)